dispatch-rules/advies-budgetwijziging.js

```diff
@@ -8,31 +7,45 @@ const rules = [];
  * -SENDER-: <http://data.lblod.info/id/representatieveOrganen/e224c637ba8bb0e5dfbb87da225b4652> Executief van de Moslims van België
  * RO: <http://data.lblod.info/id/representatieveOrganen/e224c637ba8bb0e5dfbb87da225b4652> Executief van de Moslims van België
  * CB: <http://data.lblod.info/id/centraleBesturenVanDeEredienst/e17dabfaf8f562ad181f422006c42e97> CB van de Islamitische gemeenschappen in Limburg
+ * OR
+ * -SENDER-: <http://data.lblod.info/id/representatieveOrganen/e224c637ba8bb0e5dfbb87da225b4652> Executief van de Moslims van België
+ * RO: <http://data.lblod.info/id/representatieveOrganen/e224c637ba8bb0e5dfbb87da225b4652> Executief van de Moslims van België
+ * EB: <http://data.lblod.info/id/besturenVanDeEredienst/6375F8724B5FEAF28DEDE821> Attaqwa
 **/
 let rule = {
-  documentType: 'https://data.vlaanderen.be/id/concept/BesluitType/2b12630f-8c4e-40a4-8a61-a0c45621a1e6', // Avies budgetwijziging
+  documentType: 'https://data.vlaanderen.be/id/concept/BesluitType/2b12630f-8c4e-40a4-8a61-a0c45621a1e6', // Avies budgetwijziging - EB if EB without CB or CB if EB has CB
   matchSentByEenheidClass: eenheidClass =>
     eenheidClass == 'http://data.vlaanderen.be/id/concept/BestuurseenheidClassificatieCode/36372fad-0358-499c-a4e3-f412d2eae213', // RO
   destinationInfoQuery: ( sender, submission ) => {
     return `
       PREFIX mu: <http://mu.semte.ch/vocabularies/core/>
+      PREFIX meb: <http://rdf.myexperiment.org/ontologies/base/>
       PREFIX org: <http://www.w3.org/ns/org#>
       PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
+      PREFIX ere: <http://data.lblod.info/vocabularies/erediensten/>
+      PREFIX besluit: <http://data.vlaanderen.be/ns/besluit#>
+      PREFIX eli: <http://data.europa.eu/eli/ontology#>
+      PREFIX pav: <http://purl.org/pav/>
+      PREFIX prov: <http://www.w3.org/ns/prov#>
 
       SELECT DISTINCT ?bestuurseenheid ?uuid ?label WHERE {
         BIND(${sparqlEscapeUri(sender)} as ?sender)
         BIND(${sparqlEscapeUri(submission)} as ?submission)
 
-        ?submission a <http://rdf.myexperiment.org/ontologies/base/Submission>;
-          <http://purl.org/pav/createdBy> ?sender;
-          <http://www.w3.org/ns/prov#generated> ?formData.
+        ?submission a meb:Submission;
+          pav:createdBy ?sender;
+          prov:generated ?formData.
 
         ?formData a <http://lblod.data.gift/vocabularies/automatische-melding/FormData>;
-          <http://data.europa.eu/eli/ontology#is_about> ?aboutEenheid.
+          eli:is_about ?aboutEenheid.      
+
+        OPTIONAL {
+          ?centraalBestuurVanDeEredienst org:hasSubOrganization ?aboutEenheid.
+        }
 
         VALUES ?worshipType {
-          <http://data.lblod.info/vocabularies/erediensten/CentraalBestuurVanDeEredienst>
-          <http://data.lblod.info/vocabularies/erediensten/BestuurVanDeEredienst>
+          ere:CentraalBestuurVanDeEredienst
```

Comment:
I've commented on the ticket where I'm not sure if the field “Betreffend bestuur van de eredienst” should also display CKB. Since I'm not sure I didn't trim this part of the query along with the ?worshipType section. This should work like expected anyway.

Reply:
Resolved : see https://github.com/lblod/worship-submissions-graph-dispatcher-service/pull/14/commits/a67e7976edffe031908016d26b077f4039bd696b